api.md

```diff
@@ -840,7 +846,150 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
 ---
+## Continuous Aggregates :community_function: [](continuous-aggregates)
+TimescaleDB allows users the ability to automatically recompute aggregates 
+at predefined intervals and materialize the results. This is suitable for 
+frequently used queries.
+
+> - [CREATE VIEW](#cagg_create_view)
+> - [ALTER VIEW](#cagg_alter_view)
+> - [REFRESH](#cagg_refresh_view)
+> - [DROP VIEW](#cagg_drop_view)
+
+## CREATE VIEW [](cagg_create_view) 
+`CREATE VIEW` statement is used to create continuous aggregates.
+
+The syntax is:
+``` sql
+    CREATE VIEW <view_name> [ (column_name [, ...] ) ] 
+    WITH ( timescaledb.continuous [, timescaledb.<option> = <value>])
+    AS
+    <select-query>
+    
+    <select_query> is of the form :
+    SELECT <grouping-cols>, <aggregate-functions>
```

Comment:
underscores instead of dashes.  Here and below.

Reply:
changed to underscores.